openhtf/io/output/mfg_inspector.py

```diff
@@ -196,7 +198,17 @@ def _ExtractParameters(record, testrun, used_parameter_names):
           else:
             testrun_param.description += '\nValidator: ' + str(validator)
       else:
-        _MangleMeasurement(name, value, measurement, mangled_parameters)
+        attachment = testrun.info_parameters.add()
+        attachment.name = 'multidim_%s' % name
```

Comment:
From what I can tell, attachments aren't dealt with that way, but just in case I combined all the name uniqueness mangling into one function and called that here too